src/services/pegout-status/pegout-status-rules-services.ts

```diff
@@ -0,0 +1,57 @@
+import { getLogger, Logger } from 'log4js';
+import { Log } from '../../models/rsk/log.model';
+import { RskTransaction } from '../../models/rsk/rsk-transaction.model';
+import { PegoutStatusDataModel, PegoutStatus } from '../../models/rsk/pegout-status-data.model';
+import { BRIDGE_EVENTS, getBridgeSignature } from '../../utils/bridge-utils';
+
+export class PegoutStatusRulesService {
```

Comment:
I don't understand the need for this service.
I think we should focus on trying to parse this transaction as a pegout-request-transaction if we detect it should be one (i.e. fulfills the emptyDataFilter).
Then we should run all the steps:
1. search for RELEASE_REQUEST_RECEIVED event as you do
1.1. search for RELEASE_REQUEST_REJECTED
1.1.1. set the transaction as INVALID (it should never happen)
2. decode the tx and its event data to get all the information required to register the status (e.g. amount in weis, sender address, btc receipient address, etc)